server/sonar-web/src/main/js/apps/marketplace/components/__tests__/LicenseEditionSet-test.tsx

```diff
@@ -56,9 +56,49 @@ it('should display correctly', () => {
 });
 
 it('should correctly display status message after checking license', async () => {
-  await testLicenseStatus('NO_INSTALL');
-  await testLicenseStatus('AUTOMATIC_INSTALL');
-  await testLicenseStatus('MANUAL_INSTALL');
+  let wrapper = await testLicenseStatus('NO_INSTALL', jest.fn());
+  expect(wrapper.find('p.alert')).toMatchSnapshot();
+  wrapper = await testLicenseStatus('AUTOMATIC_INSTALL', jest.fn());
+  expect(wrapper.find('p.alert')).toMatchSnapshot();
+  wrapper = await testLicenseStatus('MANUAL_INSTALL', jest.fn());
+  expect(wrapper.find('p.alert')).toMatchSnapshot();
+});
+
+it('should display terms of license checkbox', async () => {
+  let updateLicense = jest.fn();
+  let wrapper = await testLicenseStatus('NO_INSTALL', updateLicense);
+  expect(
+    wrapper
+      .findWhere(
+        n => n.type() === 'a' && n.prop('href').includes('SonarSource_Terms_And_Conditions')
```

Comment:
could you add a class or another attribute to this link to simplify the condition?